decentraland/components/presentational/LandForSaleInfo.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { StyleSheet, View, Text, Alert } from "react-native";
+import PropTypes from "prop-types";
+import {
+  responsiveWidth,
+  responsiveFontSize,
+} from "react-native-responsive-dimensions";
+import Colors from "@constants/Colors";
+import { Button, Icon } from "native-base";
+import { formatNumber } from "../../helpers";
+
+const onPriceInfo = () => {
+  const title = "";
+  const message =
+    "MANA is Decentraland’s fungible, ERC20 cryptocurrency token. MANA is burned or spent in exchange for LAND parcels. ";
+  Alert.alert(title, message, [{ text: "Okay" }]);
+};
+
+export default function LandForSale({ landForSale }) {
```

Comment:
Maybe we shouldn't call the component `LandForSale` since the file is called `LandForSaleInfo`

Reply:
Sure! Just fixed.

Reply:
Thanks! Resolving